protos/feast/core/Runner.proto

```diff
@@ -0,0 +1,73 @@
+//
+// * Copyright 2020 The Feast Authors
+// *
+// * Licensed under the Apache License, Version 2.0 (the "License");
+// * you may not use this file except in compliance with the License.
+// * You may obtain a copy of the License at
+// *
+// *     https://www.apache.org/licenses/LICENSE-2.0
+// *
+// * Unless required by applicable law or agreed to in writing, software
+// * distributed under the License is distributed on an "AS IS" BASIS,
+// * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// * See the License for the specific language governing permissions and
+// * limitations under the License.
+//
+
+syntax = "proto3";
+package feast.core;
+
+option java_package = "feast.core";
+option java_outer_classname = "RunnerProto";
+option go_package = "github.com/gojek/feast/sdk/go/protos/feast/core";
+
+message DirectRunnerConfigOptions {
```

Comment:
Is there a reason why runner configuration is defined as a proto? Not that its wrong -- it's just strange that it's the only piece of application configuration that is

Reply:
So my rough thoughts on this are:
- If the configuration is defined clearly in FeastProperties then we probably want to make that the source of truth. 
- If the configuration in FeastProperties contains a generic container like `Map<String, String>` then we need to somehow document that. For validation I want to delegate that to the implementation layer like to the stores themselves. So the user would go to the stores documentation.
- For runners we are also using a `Map<String, String>` and we have a validation class, but its not really easy to find. Ideally we want documentation to point to.

So the easiest way to document this is to define it through a proto, because the proto will generate documentation that we can point users to. We are already doing this for `Source` and `Stores`, so I think adding runners is natural for now. The options will show up here: https://api.docs.feast.dev/grpc/feast.core.pb.html

Reply:
ok